src/application_config/app_config.h

```diff
@@ -1,8 +1,37 @@
+/**
+ * @file app_config.h
+ * @author Otso Jousimaa <otso@ojousima.net>
+ * @date 2020-04-16
+ * @brief Master configuration base
+ *       Variables which can be adjusted by application_mode_xxxxx.h are preceeded by #ifdef
+ *       2021-12-10 DG12 correct _FLASH_PAGES from 16 to 0x15000/4096-1 see loader configuration.
```

Comment:
History is maintained in git, not in header comments. Please remove the duplicate information

Reply:
Please show me how to extract meaningful comments from git.

Git has pull level comments. Not file level and not function level.

How do I (as some who is not the author ) understand  a comment like <pre>"Fix #308 - off buy one error in log blocks #320"</pre>

Do you review the history in git when you begin to look at file to add a feature or try to locate a potential  problem?
The information I provided is brief and easily available and understandable.

<small>(PS You should  change the. "buy" to "by" )</small>

Reply:
It's possible to list changes to a file with git: https://stackoverflow.com/questions/278192/view-the-change-history-of-a-file-using-git-versioning

If the commit history of git is not clear enough, the proper fix is to have better discipline in making commits.